pbft/support/scripts/build.sh

```diff
@@ -19,6 +26,7 @@ fi
 
 LDFLAGS="-s -w -X common.GitCommitHash=$(git_commit_hash)"
 
-go build -o "$OUTDIR"/truechain-engine \
+go build -o "$OUTDIR"/pbft-client \
```

Comment:
Hey so `pbft-client` is essentially doing the same thing as `truechain-engine` and we're not building `pbft-server` anywhere in the build scripts.. While what we want is for client to just support to use .proto files to form requests and send txns / receive blocks.. (the block structure though, since it's not defined yet in https://github.com/truechain/py-trueconsensus, is still fine) 
but mainly what we could now cover is also the server's daemon'ized part coupled with manual client.go triggers and eventually get rid of this hardcoded numquests value from https://github.com/truechain/truechain-consensus-core/pull/14

Reply:
Correct. The servers listeners currently running as goroutines listen to grpc requests from clients through 'grpc ports'. The next steps would be to work on the server daemonization part and getting rid of the hardcoded cfg.numQuest variable.